docs/notebooks/cold_plasma_tensor_elements.ipynb

```diff
@@ -49,248 +58,536 @@
+     "name": "#%%\n"
+    }
    },
+   "outputs": [],
    "source": [
-    "Take a look at the docs to :func:`~plasmapy.formulary.dielectric.cold_plasma_permittivity_SDP` and :func:`~plasmapy.formulary.dielectric.cold_plasma_permittivity_LRP` for more information on these."
+    "B = 2 * u.T\n",
+    "species = [\"e\", \"D+\"]\n",
+    "n = [1e18 * u.m**-3, 1e18 * u.m**-3]"
    ]
   },
   {
    "cell_type": "markdown",
-   "metadata": {},
+   "metadata": {
+    "pycharm": {
+     "name": "#%% raw\n"
+    }
+   },
    "source": [
-    "Let's define some parameters, such as the magnetic field magnitude,\n",
-    "the plasma species and densities and the frequency band of interest\n",
-    "\n"
+    "Let's pick some frequencies in the radio frequency range."
    ]
   },
   {
    "cell_type": "code",
    "execution_count": null,
    "metadata": {
-    "collapsed": false,
-    "jupyter": {
-     "outputs_hidden": false
+    "pycharm": {
+     "name": "#%%\n"
     }
    },
    "outputs": [],
    "source": [
-    "B = 2 * u.T\n",
-    "species = [\"e\", \"D+\"]\n",
-    "n = [1e18 * u.m**-3, 1e18 * u.m**-3]\n",
+    "f_min = 1 * u.MHz\n",
+    "f_max = 200 * u.GHz\n",
+    "\n",
+    "f = np.geomspace(f_min, f_max, num=5000).to(u.Hz)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "pycharm": {
+     "name": "#%% raw\n"
+    }
+   },
+   "source": [
+    "[equivalency]: https://docs.astropy.org/en/stable/units/equivalencies.html\n",
+    "[angular frequencies]: https://docs.plasmapy.org/en/stable/development/code_guide.html?highlight=angular%20frequencies#angular-frequencies\n",
     "\n",
-    "f = np.logspace(start=6, stop=11.3, num=3001)  # 1 MHz to 200 GHz\n",
-    "omega_RF = f * (2 * np.pi) * (u.rad / u.s)"
+    "Next we convert these frequencies to angular frequencies. To do this we must specify an [equivalency] between a cycle per second and a hertz. The reasons why are described in the section of PlasmaPy's documentation on [angular frequencies]."
    ]
   },
   {
    "cell_type": "code",
    "execution_count": null,
    "metadata": {
-    "collapsed": false,
-    "jupyter": {
-     "outputs_hidden": false
+    "pycharm": {
+     "name": "#%%\n"
     }
    },
    "outputs": [],
    "source": [
-    "help(cold_plasma_permittivity_SDP)"
+    "ω_RF = f.to(u.rad / u.s, equivalencies=[(u.Hz, u.cycle / u.s)])"
+   ]
+  },
+  {
+   "cell_type": "raw",
+   "metadata": {
+    "pycharm": {
+     "name": "#%% raw\n"
+    }
+   },
+   "source": [
+    "In a Jupyter notebook, we can type `\\omega` and press tab to get \"ω\"."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "pycharm": {
+     "name": "#%% raw\n"
+    }
+   },
+   "source": [
+    "[Stix 1992]: https://link.springer.com/book/9780883188590\n",
+    "\n",
+    "Now we are ready to calculate the $S$ (sum), $D$ (difference), and $P$ (plasma) components of the dielectric tensor in the \"Stix\" frame with $B ∥ ẑ$.  This notation is from [Stix 1992]."
```

Comment:
These two lines are be rendered as one paragraph jammed together.

![image](https://user-images.githubusercontent.com/29869348/176553987-f4f5ad21-aee3-4337-aaaf-77fcf13867f1.png)

I'd say just put them in the same cell and put a blank line between them.

Reply:
Good catch!  I think this may have happened because the first cell was in the raw nbconvert format, and the second cell was in Markdown.  In any case, I'll merge the two cells.